src/libcrun/container.c

```diff
@@ -1548,47 +1548,52 @@ container_delete_internal (libcrun_context_t *context, runtime_spec_schema_confi
       if (UNLIKELY (ret < 0))
         return ret;
       if (ret == 1)
-        return crun_make_error (err, 0, "the container `%s` is not in 'stopped' state", id);
+        {
+
+          ret = libcrun_status_has_read_exec_fifo (state_root, id, err);
+          if (ret <= 0)
+            return crun_make_error (err, 0, "the container `%s` is not in `stopped` or `created` state", id);
+
+          /* If the container is in "created" state, then do the equivalent of delete --force.  */
```

Comment:
nit, might want to add ' or "stopped" ' after "created" given the output to the user.

Reply:
we end up here only when the container is running (so in the `created` state)